BUILDING.md

```diff
@@ -12,4 +12,18 @@ repository.
 
 Prior to publishing, you'll need to login to [Docker Hub][DH] using the `docker login` command.
 
+## Updating and adding versions
+
+The `update.sh` script will update all the Dockerfiles in the versioned folder directories to the latest JanusGraph release matching the corresponding version
+
+```bash
+./update.sh
+```
+
+An optional argument can be provided to create a new versioned folder directory. There must be an existing JanusGraph release matching the provided version.
+
+```bash
+./update.sh 0.4
+```
+
```

Comment:
You could also add a sentence that explains that resources and the `Dockerfile` will be copied over from `build` to the version directory.